### DISCUSSION TITLE
Internal Server Error

### DISCUSSION THREAD

**Discussion by Crysis123467** [NethercraftMC5608/NetherWorkshopDownloader#18] [Q&A] [answered]
Need Help, Can't download Mods, First It Shows Nothing, Then i am always getting this error
![Screenshot (5241)](https://github.com/NethercraftMC5608/NetherWorkshopDownloader/assets/128551109/ae753c47-5b92-4090-99bc-febf73ce811c)

**Comment by NethercraftMC5608** [comment, marked as answer]
I need to update the release.  It is because my domain expired. I will post when I create the new release👍

**Comment by Crysis123467** [comment]
can you please increase the download size limit, it will be very helpful.
